lib/nice_partials/monkey_patch.rb

```diff
@@ -24,4 +24,17 @@ def with_nice_partials_t_prefix(lookup_context, block)
   end
 end
 
+module NicePartials::RenderingWithAutoContext
+  attr_reader :content
+
+  def render(options = {}, locals = {}, &block)
+    _content = content
+    _layout_for(@content = np) if block&.arity == 1 # Mimic standard `yield` by calling into `_layout_for` directly.
```

Comment:
I _think_ this works?

```suggestion
    _layout_for(@content = np, &block) if block&.arity == 1 # Mimic standard `yield` by calling into `_layout_for` directly.
```

Reply:
Ah you're right, yes! I have no idea how the tests passed yesterday then, because they were also failing for me locally.